index.html

```diff
@@ -1,3 +1,134 @@
-<html>
-  <h1>hello main</h1>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>my Portfolio</title>
+</head>
+<body>
+    <header>
+        <a href="#" class="logo-icon">Logo</a>
+        <img src="img/Union@2x.png" alt="logo icon" class="logo-icon">
+    </header>
+    <main>
+        <div class="text-section">
+            <h1 class="text1">I’m Bunbee <br>Glad to see you!</h1>
+            <p class="text2">
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <p>LET'S CONNECT</p>
+            <ul class="footer-section">
+                <li><a href="#"><img src="img/github@2x.png" alt="github.com"></a></li>
+                <li><a href="#"><img src="img/Linkedinicon@2x.png" alt="Linkedin"></a></li>
+                <li><a href="#"><img src="img/twitter@2x.png" alt="twitter"></a></li>
+                <li><a href="#"><img src="img/medium@2x.png" alt="Medium"></a></li>
+            </ul>
+        </div><br><br><br>
+            
```

Comment:
- [ ] I think it's better to use HTML semantic tag `<section>` to wrap my work section with a display grip to position all the project cards. It will make your webpage more semantically correct.